racunaljka.js

```diff
@@ -0,0 +1,230 @@
+/*=================================== brojevi==================================================*/
```

Comment:
Additional note: Setup your editor that it uses spaces instead of tabs for indentation.